app/views/account/admin/billing_contracts/show.html.slim

```diff
@@ -16,6 +16,6 @@
         | Actions
       nav
         ul.nav.navbar
-          li.nav-item = link_to 'Edit contract', edit_billing_contract_path, class: "btn btn-outline-warning"
-          li.nav-item = link_to 'Delete contract',  @billing_contract, method: :delete, data: { confirm: 'Are you sure?' }, class: "btn btn-outline-danger"
-          li.nav-item = link_to 'Back to contracts', billing_contracts_path, class: "btn btn-outline-info"
+          li.nav-item = link_to 'Edit contract', edit_account_admin_billing_contract_path, class: "btn btn-outline-warning"
```

Comment:
you were saying about long _path helpers
you can just put newlines after each comma to make it more readable